core/pom.xml

```diff
@@ -81,6 +81,13 @@
             <version>${json.org.version}</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>info.debatty</groupId>
+            <artifactId>java-string-similarity</artifactId>
```

Comment:
I see a small disadvantages for e.g. iOS to have this in a separate jar just for the sake of one class. Can we just copy this class (and remove the two interfaces) to a package like we did for apache commons (plus add this in the NOTICE for proper attribution)? Also then we could remove the Levenshtein class?